server/session/handler_book_edit.go

```diff
@@ -0,0 +1,65 @@
+package session
+
+import (
+	"fmt"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/sandertv/gophertunnel/minecraft/protocol/packet"
+	"golang.org/x/exp/slices"
+)
+
+// BookEditHandler handles the BookEdit Packet
+type BookEditHandler struct{}
+
+// Handle ...
+func (b BookEditHandler) Handle(p packet.Packet, s *Session) error {
```

Comment:
Make sure you look this logic over; just tested for a few minutes in game and got disconnected for:
```
failed processing packet from 127.0.0.1:54647 (JustTalz): *packet.BookEdit: may only come before a page which already exists
```